src/BizHawk.Client.EmuHawk/tools/Watch/RamWatch.cs

```diff
@@ -350,14 +350,6 @@ private void EditWatch(bool duplicate = false)
 
 			if (SelectedWatches.Any())
 			{
-				foreach (var sw in SelectedWatches)

```

Comment:
The reasoning behind this check was range considerations.  If you have a nes wram and system bus domain, and add 0x1000 for isntance, that's an invalid wram address but valid system bus.  So to not let the user add out of range addresses we restricted to a single comain

Reply:
After some thought and discusison, we agreed this is okay.  There isn't much you can do in the editor with multiple watches anyway (addresses is grayed out).  And we gracefully handle out of range addresses anyway.